src/cache.cpp

```diff
@@ -620,6 +620,9 @@ bool convert_header_to_stat(const char* path, headers_t& meta, struct stat* pst,
   // mtime
   pst->st_mtime = get_mtime(meta);
 
+  // ctime
+  pst->st_ctime = get_ctime(meta);
```

Comment:
Calling this ctime seems wrong which should be metadata change time.  Instead call it btime for file birth?